assets/styles/index.scss

```diff
@@ -6,6 +11,3 @@
 @import 'fonts';
 @import 'typography';
 @import 'links';
-
-@import 'components/header';
-@import 'components/footer';
```

Comment:
Does this mean that header and footer need to be manually included now?

Reply:
Yeah, thought it might be more consistent not to include any components by default.

Reply:
🤔